sqlsynthgen/make.py

```diff
@@ -270,5 +273,17 @@ def make_src_stats(
             )
             private_result = reader.execute(stat_data["query"])
             # The first entry in the list names the columns, skip that.
-            src_stats[stat_data["name"]] = private_result[1:]
+            return private_result[1:]
+
+    else:
+
+        def execute_query(conn: Any, stat_data: Dict[str, Any]) -> Any:
```

Comment:
Do you think we should type the return as List[List] (or Iterable[Iterable]) or similar?

```suggestion
        def execute_query(conn: Any, stat_data: Dict[str, Any]) -> List[List]:
```

Reply:
The problem is we need to change the type signature of `execute_query` in both branches of the if statement to keep mypy happy (`All conditional function variants must have identical signatures`), and mypy doesn't know what the return type of SNSQL's `private_reader.execute` is.